### PR TITLE
Calendar A11Y Improvements

### LINKED ISSUE #9194: Calendar button titles and "Go to today" focus
This issue relates to a subset of [Calendar A11y Pass #8349](https://github.com/OfficeDev/office-ui-fabric-react/issues/8349) (specifically "Bug1" and "Bug2")

## Codepen demonstrating the issue
[Standard Calendar](https://codepen.io/anon/pen/zQRmRw?&editable=true)

## Actual behavior:
* Icon buttons for next/prev month should include tooltips for users who don't use a screenreader
* Focus is lost when GoToToday is pressed

## Expected behavior:
* Tooltip is present on icons without other labels
* Focus returns to current day when GoToToday is pressed

## Documentation describing expected behavior

The desired behavior was flagged by A11Y auditors.
 [name computation](https://www.w3.org/TR/wai-aria-1.1/#namecalculation) or purpose of [aria-label](https://www.w3.org/TR/wai-aria-1.1/#aria-label).

### PR DESCRIPTION
**Pull request by cjsheets**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #9194 
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

Add title attributes to prev/next icon buttons in the Calendar component. Improve focus handling after "Go to today" is clicked.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9195)

### CODE REVIEW

**Comment by JasonGore** [comment]
@lorejoh12 Hey John could you double-check this one? Thanks!

File: packages/office-ui-fabric-react/src/components/Calendar/CalendarMonth.tsx [hunk 1]
```diff
@@ -168,6 +169,7 @@ export class CalendarMonth extends BaseComponent<ICalendarMonthProps, ICalendarM
                     ? strings.nextYearAriaLabel + ' ' + dateTimeFormatter.formatYear(addYears(navigatedDate, 1))
                     : undefined
                 }
+                title={strings.nextYearAriaLabel ? strings.nextYearAriaLabel : undefined}
```
Comment (JasonGore) [inline packages/office-ui-fabric-react/src/components/Calendar/CalendarMonth.tsx]: The typing already constrains this type to `string | undefined`, so I think the ternary check is redundant.

Comment (cjsheets) [reply packages/office-ui-fabric-react/src/components/Calendar/CalendarMonth.tsx]: I agree, I updated the PR with the ternary removed.

**Comment by cjsheets** [comment]
Nice catch, I didn't notice CalendarYear. I added tooltips to those buttons as well.

**Comment by cjsheets** [comment]
It looks like someone else merged a fix for the tooltips 😊 Now this PR is just the focus fix.

**Pull request merged**